{"name": "登录失败", "status": "passed", "attachments": [{"name": "log", "source": "ef5632d6-125b-4c56-92f5-a10b74400ff4-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "cases", "value": "{'mobile': '1303845892@qq', 'pwd': 'nmb_python', 'expected': '用户不存在'}"}], "start": 1614337817073, "stop": 1614337818399, "uuid": "79d74183-55cb-4f6c-9e55-2af68b53d754", "historyId": "2752b8ba2a2c671bdc0b83f505e377c1", "testCaseId": "6a7e4d99881775cbe707054555e1e02d", "fullName": "testcases.test_1_login.TestLogin#test_login_error", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_1_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "chnegbo"}, {"name": "thread", "value": "20996-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_1_login"}]}